src/components/Settings/integrations/Okta/OktaAccordion.tsx

```diff
@@ -3,33 +3,33 @@ import { useTranslation } from 'react-i18next';
 import { AccordionItem } from 'src/components/Shared/Forms/Accordions/AccordionItem';
 import { FieldWrapper } from 'src/components/Shared/Forms/FieldWrapper';
 import { AccordionProps } from '../integrationsHelper';
-import { useGetKeyAccountsQuery } from './Key.generated';
+import { useOktaAccountsQuery } from './OktaAccounts.generated';
 
-export const TheKeyAccordion: React.FC<AccordionProps> = ({
+export const OktaAccordion: React.FC<AccordionProps> = ({
   handleAccordionChange,
   expandedPanel,
   disabled,
 }) => {
   const { t } = useTranslation();
-  const { data, loading } = useGetKeyAccountsQuery();
-  const keyAccounts = data?.user?.keyAccounts;
+  const { data, loading } = useOktaAccountsQuery();
+  const oktaAccounts = data?.user?.keyAccounts;
   return (
     <AccordionItem
       onAccordionChange={handleAccordionChange}
       expandedPanel={expandedPanel}
-      label={t('The Key / Relay')}
+      label={t('Okta')}
       value={''}
       disabled={disabled}
       image={
         <img
-          src="/images/settings-preferences-intergrations-key.png"
-          alt="The Key"
+          src="/images/settings-preferences-intergrations-okta.png"
```

Comment:
Can we delete `settings-preferences-intergrations-key.png` now?

FYI, there's a typo in the name `intergrations` --> `integrations`, but all the other images have the typo too.